scripts/analyze-submission

```diff
@@ -66,10 +70,48 @@ class AnalyzeSubmission:
             for uuid in [k for (k, v) in self.bundle_map.items() if v['type'] == 'primary']:
                 yield uuid
 
+        @property
+        def analysis_workflow_count(self):
+            """Return the number of the analysis workflows from bundle_map, if the map is empty, return 0."""
+            return sum([len(v['analysis_workflows']) for (k, v) in self.bundle_map.items() if v['type'] == 'primary'])
+
+        @property
+        def succeeded_analysis_workflow_count(self):
+            """Return the number of succeeded analysis workflows from bundle_map, if the map is empty, return 0."""
+            return sum([wf_body['status'] == 'Succeeded' for (k, v) in self.bundle_map.items()
+                        if v['type'] == 'primary' for wf_id, wf_body in v['analysis_workflows'].items()])
+
+        @property
+        def failed_analysis_workflow_count(self):
+            """Return the number of failed or aborted analysis workflows from bundle_map,
+                if the map is empty, return 0.
+            """
+            return sum([wf_body['status'] in ('Failed', 'Aborted') for (k, v) in self.bundle_map.items()
+                        if v['type'] == 'primary' for wf_id, wf_body in v['analysis_workflows'].items()])
+
+        @property
+        def running_analysis_workflow_count(self):
+            """Return the number of ongoing analysis workflows from bundle_map, if the map is empty, return 0."""
+            return sum([wf_body['status'] not in ('Failed', 'Aborted', 'Succeeded')
```

Comment:
Doing a more explicit check that `wf_body['status'] in ('Running', 'Aborting')` would be clearer here. Also where do you think On Hold workflows should go? Right now they will be reflected in this count of running workflows.

Reply:
Done renaming

Reply:
I'm about to merging this PR, let's try to improve this in a followup PR if it's still not optimal?

Reply:
Ok, I see what you mean! My reasoning was that users would need to know the possible Cromwell statuses to know what is included in "Running" workflows but with a broader interpretation that it refers to anything "in progress" the specific status details may not be too important.

Reply:
@samanehsan Users will still be able to see the detailed exact statuses in the saved result JSON file, but will only see this broader "in progress" in the command line output, not sure if this makes sense!